benchmark/sampling/neighborloader_bench.py

```diff
@@ -0,0 +1,105 @@
+import argparse
+import os.path as osp
+from timeit import default_timer
+
+from ogb.nodeproppred import PygNodePropPredDataset
+
+import torch_geometric.transforms as T
+from torch_geometric.datasets import OGB_MAG
+from torch_geometric.loader import NeighborLoader
+
+
+def run(args: argparse.ArgumentParser) -> None:
+
+    print("BENCHMARK STARTS")
+    for dataset_name in args.datasets:
+        print("Dataset: ", dataset_name)
+
+        if args.datasets_root == 'data':
```

Comment:
Shouldn't we set `args.datasets_root` to `../../data` by default? Moving this two layers up looks a bit unexpected to me. We can also just leave in `root_dir = "data"` though.

Reply:
Done - i left only root with "../../data" to not overcomplicate it. Other paths can be handle by redirecting by root.